openllmtelemetry/instrument.py

```diff
@@ -1,13 +1,15 @@
+import hashlib
 import os
 from logging import getLogger
-from typing import Dict, Optional, Tuple
+from typing import Dict, List, Optional, Tuple
```

Comment:
List is unused, since you annotate below with list, right?

Reply:
Good catch, fixed!